util/stringutil/string_util.go

```diff
@@ -236,16 +236,25 @@ func DoMatch(str string, patChars, patTypes []byte) bool {
 	return sIdx == len(str)
 }
 
-// RemoveBlanks removes all blanks, returns a new string.
-func RemoveBlanks(s string) string {
+// RemoveRedundantBlanks removes all redundant blanks, returns a new string.
+func RemoveRedundantBlanks(s string) string {
```

Comment:
Could you add an unit test for this?

Reply:
There are unittests in `util/stringutil/string_util_test.go`.